config/resources/defaults.yaml

```diff
@@ -17,6 +17,18 @@
 Debug: false
 Logging:
   Level: "Error"
+  Cache:
+    Scitokens: error
+    XrootdPssSetOptCache: error
+    Pss: error
```

Comment:
I was thinking of having `Logging.Cache.Pss` control both `pss.trace` and `pss.setopt` for the cache. Also it looks like you're missing this from the origin?